packages/code-snippet-block/package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "code-snippet-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli block serve --entryPath src/index.ts",
+        "deploy": "frontify-cli block deploy --entryPath src/index.ts",
+        "lint": "eslint src",
+        "lint:fix": "eslint --fix src",
+        "prettier": "prettier --check src",
+        "prettier:fix": "prettier --write src",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "7.17.10",
+        "@cypress/react": "5.12.4",
+        "@frontify/eslint-config-react": "0.15.2",
+        "@frontify/frontify-cli": "5.1.1",
+        "@types/react": "17.0.44",
+        "autoprefixer": "10.4.6",
+        "cypress": "9.6.0",
+        "eslint": "8.15.0",
+        "postcss": "8.4.13",
+        "prettier": "2.6.2",
+        "tailwindcss": "3.0.24",
+        "typescript": "4.6.4"
+    },
+    "dependencies": {
+        "@codemirror/autocomplete": "^0.19.0",
+        "@codemirror/closebrackets": "^0.19.0",
+        "@codemirror/commands": "^0.19.8",
+        "@codemirror/comment": "^0.19.0",
+        "@codemirror/fold": "^0.19.3",
+        "@codemirror/gutter": "^0.19.9",
+        "@codemirror/highlight": "^0.19.7",
+        "@codemirror/history": "^0.19.2",
+        "@codemirror/lang-css": "^0.19.3",
+        "@codemirror/lang-html": "^0.19.4",
+        "@codemirror/lang-javascript": "^0.19.7",
+        "@codemirror/lang-json": "^0.19.2",
+        "@codemirror/lang-php": "^0.19.1",
+        "@codemirror/language": "^0.19.1",
+        "@codemirror/lint": "^0.19.0",
+        "@codemirror/matchbrackets": "^0.19.4",
+        "@codemirror/panel": "^0.19.1",
+        "@codemirror/rectangular-selection": "^0.19.1",
+        "@codemirror/search": "^0.19.8",
+        "@codemirror/state": "^0.19.8",
+        "@codemirror/text": "^0.19.6",
+        "@codemirror/view": "^0.19.42",
+        "@frontify/app-bridge": "3.0.0-beta.11",
+        "@frontify/fondue": "9.9.0",
```

Comment:
could you update it to v.11? The color format changed there from `{r: ..}` to `{red: ..}` and icons are named differently. See [changelog](https://github.com/Frontify/fondue/releases/tag/v11.0.0)